src/codegen/function_body_chunk.rs

```diff
@@ -1080,17 +1076,15 @@ impl Builder {
                 ref mem_mode,
             } = *par
             {
-                if self
-                    .transformations
-                    .iter()
-                    .any(|tr| match tr.transformation_type {
+                if self.transformations.iter().any(|tr| {
```

Comment:
Clippy complains about this still, if you don't want to change it can you add an `#[allow(clippy::...)]` in the right place?

Reply:
It doesn't on nightly, so...